### PR TITLE
add rgu tracker

### PR DESCRIPTION
**Pull request by Jachryl**
Adds RGU Tracker and updated Readme

### CODE REVIEW

File: src/Jackett.Common/Definitions/rgu.yml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      default: For best results, change the 'Torrents per page' setting to 100 on your profile.
+      
+  login:
+    path: login.php
+    method: form
+    form: form[action="takelogin.php"]
+    inputs:
+      username: "{{ .Config.username }}"
+      password: "{{ .Config.password }}"
+      submitme: "X"
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/rgu.yml]: I should think that only one of these submitme is required

File: src/Jackett.Common/Definitions/rgu.yml [hunk 2]
```diff
@@ -0,0 +1,147 @@
+---
+  site: RGU
+  name: RGU
+  description: "RGU is a Private site for MOVIES / TV / GENERAL"
+  language: en-us
+  type: private
+  encoding: UTF-8
+  links:
+    - https://rgu.rgt.life
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/rgu.yml]: please add a forward slash at the end of the links `https://rgu.rgt.life/`

**Comment by kaso17** [comment]
added /, merged

**Comment by Jachryl** [comment]
I need to add the second  submitme: "X"  back in for it to work...

**Comment by kaso17** [comment]
That shouldn't make any difference but I added it back via ccd5347be2bce2d9e511eee81be65d2f33579e5a

**Comment by garfield69** [comment]
v2.rgut.uk returning ERR_NAME_NOT_RESOLVED
just me, or is RGU broken, gone or moved to a new domain?

**Comment by garfield69** [comment]
https://v2.rgut.uk/ returning ERR_NAME_NOT_RESOLVED
anyone have any news?
@mstrdraco ?

**Comment by garfield69** [comment]
@Jachryl v2.rgut.uk returning ERR_NAME_NOT_RESOLVED
just me, or is RGU broken, gone or moved to a new domain?

**Comment by Jachryl** [comment]
It is no longer an active tracker.

-J

**Comment by garfield69** [comment]
thanks. i'll remove it in the next update cycle.

**Pull request merged**